addons/binding/org.openhab.binding.coolmasternet/src/main/java/org/openhab/binding/coolmasternet/handler/HVACHandler.java

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.coolmasternet.handler;
+
+import static org.openhab.binding.coolmasternet.config.coolmasternetConfiguration.*;
+import static org.openhab.binding.coolmasternet.coolmasternetBindingConstants.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.ThingUID;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.coolmasternet.internal.ControllerHandler;
+import org.openhab.binding.coolmasternet.internal.ControllerHandler.CoolMasterClientError;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link HVACHandler} is responsible for handling commands for a single
+ * HVAC unit (a single UID on a CoolMasterNet controller.)
+ *
+ * @author Two Feathers Pty Ltd - Initial contribution
```

Comment:
Could you put "Angus Gratton" in the javadocs as well (as on the manually written classes)?